examples/list-binding-pattern-in-match-statement/list_binding_pattern_in_match_statement.out

```diff
@@ -0,0 +1,6 @@
+$ bal run list_binding_pattern_in_match_statement.bal
+List all files
+Remove all files
+Remove files: ["a.bal","b.bal"]
```

Comment:
```suggestion
Remove files: [`a.bal` and `b.bal`].
```

Reply:
This output statement is generated by `io:println("Remove files: ", files);` which outputs the content of the `...files` rest binding pattern, hence it uses double quotes and cannot be changed.